### ISSUE TITLE
advanced03-AF_XDP: compile error

### ISSUE THREAD

**Issue by glennWang** [xdp-project/xdp-tutorial#83]
As the title say.

Firstly execute _make_ command in the advanced03-AF_XDP dir.

$ make

Then I will get the following error.

```
clang -S \
    -target bpf \
    -D __BPF_TRACING__ \
    -I../libbpf/src//build/usr/include/ -I../headers/ \
    -Wall \
    -Wno-unused-value \
    -Wno-pointer-sign \
    -Wno-compare-distinct-pointer-types \
    -Werror \
    -O2 -emit-llvm -c -g -o af_xdp_kern.ll af_xdp_kern.c
af_xdp_kern.c:5:10: fatal error: 'bpf_helpers.h' file not found
#include "bpf_helpers.h"
         ^~~~~~~~~~~~~~~
1 error generated.
make: *** [../common//common.mk:111: af_xdp_kern.o] Error 1
```

By the way, when I git clone from https://github.com/chaudron/xdp-tutorial , there is no problem.

**Comment by punithpatil** [comment]
From https://github.com/xdp-project/xdp-tutorial/commit/a339ed6e2d160db97e738f7b01293b05125cfea4 it looks like libbpf now includes bpf_helpers.h
Rewriting the include in the `af_xdp_kern.c` to use `<bpf/bpf_helpers.h>` should fix the problem.

**Comment by tohojo** [comment]
Fixed by c553e5b6b765dcfe3eddccdad853a1cfa50b8a3d
